ansible/pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = ""
 authors = ["jschaul <jschaul@users.noreply.github.com>"]
 
 [tool.poetry.dependencies]
-python = "^2.7 || >=3.5"
+python = ">=3.5"
```

Comment:
I think I used poetry 1.1.0b2 because 1.0.10 wasn't working. I forgot what was the reason. I will figure this out and either put a note saying you need to use poetry >=1.1 or just commit the lock with poetry 1.0.10.

Reply:
I fixed it in https://github.com/wireapp/wire-server-deploy/pull/337/commits/0c6de4965466c5870b0a29212bf2a0902a390801, not really happy with it. But we can iterate on it as we go, I guess.